File: mpp/lab3/src/main/java/domain/Arbiter.java
package domain;

public class Arbiter {
    private int id = 0;
    private final String name;
    private final String password;
    private final ScoreType type;

    public Arbiter(String name, String password, ScoreType type) {
        this.name = name;
        this.password = password;
        this.type = type;
    }

    public Arbiter(int id, String name, String password, ScoreType type) {
        this(name, password, type);

        this.id = id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }

    public ScoreType getType() {
        return type;
    }

    @Override
    public String toString() {
        return "Arbiter{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", type=" + type +
                '}';
    }
}

File: mpp/lab8/client_csharp/Domain/Arbiter.cs
using Newtonsoft.Json;

namespace client_csharp.Domain
{
    public class Arbiter
    {
        [JsonProperty("id")]
        public int? Id { get; set; }

        [JsonProperty("name")]
        public string Name { get; set; }
        
        [JsonIgnore]
        public string Password { get; set; }
        
        public override string ToString()
        {
            return $"Arbiter{{Id={Id}, Name={Name}}}";
        }
    }
}
